readme.md

```diff
@@ -1,115 +1,96 @@
-
-## The Large Test Files Script
-The Large Test Files script in [`large_test_files_script.py`](https://github.com/cfpb/hmda-test-files/blob/master/python/large_test_files_script.py) creates a test file based on existing test file data to create a new file with a larger number of rows. The script would only need to be run, and the attributes of the new file to be created can be changed with [`large_file_script_config.yaml`](https://github.com/cfpb/hmda-test-files/blob/master/python/configurations/large_file_script_config.yaml) in `python/configurations`. Files can be created with any number of rows, and can be made for any test institution so long as the Institution Name, LEI and Tax ID are provided in the configuration file. The configuration file stores filepaths for source data, as well as the filepaths for output. 
-
-In addition, the script contains functionality for changing values by column and row using a yaml configuration. An example of this yaml configuration is located in `python/configurations/` as 
-[`row_by_row_modification.yaml.`](https://github.com/cfpb/hmda-test-files/blob/master/python/configurations/row_by_row_modification.yaml) The script can contain a path to a yaml configuration to initiate this functionality. 
-
-## Getting Started: Generating Clean Files, Edit Test Files, and Large Files. 
-The function method `create_files(kind)` outputs a clean test file or a set of edit test files, with an input indicating the kinds of file to be created, `"clean_file"` or `"error_files"`. The `kind` keyword in the function is inputted as a string. 
-
-A suggested workflow for creating test files:
+## Generating Clean Files
+Clean files will pass the HMDA edits (business rules for data submission). These files are used as the base for generating files that will fail edits. Running the following scripts will create the edits_files directory and a data file that will pass the HMDA edit checks. The file will have a number of rows set in a YAML clean file configuration for each directory. Other variables, such as data ranges can also be set in the configuration files.
 
-1) *Instantiate FileGenerator*: Instantiate `FileGenerator()` to a python object in a script in the `python/` directory. Such as `file = FileGenerator()`.
+For 2019:
+1. Run `cd 2019/python` to navigate to the python directory. 
+2. Adjust the [2019 Clean File Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/clean_file_config.yaml) for bank name, LEI, Tax ID and number of rows. 
+3. Run `python3 generate_2019_clean_files.py`
+4. The clean test file should appear in a new edits_files directory under `2019/edits_files/clean_files/{Bank Name}/` with the filename `clean_file_{Number or Rows}_{Bank Name}.txt`. 
```

Comment:
Are there known cases where the clean file appears in another directory? If not, then replace `should` with `will` and `appear` with `be created`.